webapp/src/utils/tooltip_utils.ts

```diff
@@ -0,0 +1,23 @@
+export const validateGitlabURL = (url: string): boolean => {
+    const gitlabRegexPattern = /https?:\/\/(www\.)?.*\/([\w.?-]+)\/([\w-]+)\/-\/([\w-]+)\/([\d-]+$)/g;
```

Comment:
Is the issue we were experiencing on community fixed where the configured GitLab URL is `https`, but the posted link for a DM notification was `http`, so the tooltip did not open?

Reply:
@mickmister No, we are working on finding the cause of that issue.

Reply:
Honestly not sure exactly what the regex is doing. Maybe we can have a comment to show what it's specifically trying to match on? Also, we can place the regex creation outside of this function